client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -58,49 +107,81 @@ def process(self, instance):
         filename = "{name}.abc".format(**instance.data)
         path = os.path.join(parent_dir, filename)
 
-        options = {
-            "step": instance.data.get("step", 1.0),
-            "attr": attrs,
-            "attrPrefix": attr_prefixes,
-            "writeVisibility": True,
-            "writeCreases": True,
-            "writeColorSets": instance.data.get("writeColorSets", False),
-            "writeFaceSets": instance.data.get("writeFaceSets", False),
-            "uvWrite": True,
-            "selection": True,
-            "worldSpace": instance.data.get("worldSpace", True)
-        }
-
+        root = None
         if not instance.data.get("includeParentHierarchy", True):
             # Set the root nodes if we don't want to include parents
             # The roots are to be considered the ones that are the actual
             # direct members of the set
-            options["root"] = roots
+            root = roots
 
-        if int(cmds.about(version=True)) >= 2017:
-            # Since Maya 2017 alembic supports multiple uv sets - write them.
-            options["writeUVSets"] = True
+        kwargs = {
+            "file": path,
+            "attr": attrs,
+            "attrPrefix": attr_prefixes,
+            "dataFormat": attribute_values.get("dataFormat", self.dataFormat),
+            "endFrame": end,
+            "eulerFilter": attribute_values.get(
+                "eulerFilter", self.eulerFilter
+            ),
+            "noNormals": attribute_values.get("noNormals", self.noNormals),
+            "preRoll": attribute_values.get("preRoll", self.preRoll),
+            "preRollStartFrame": attribute_values.get(
+                "preRollStartFrame", self.preRollStartFrame
+            ),
+            "renderableOnly": attribute_values.get(
+                "renderableOnly", self.renderableOnly
+            ),
+            "root": root,
+            "selection": True,
+            "startFrame": start,
+            "step": instance.data.get(
+                "creator_attributes", {}
+            ).get("step", 1.0), #missing
```

Comment:
```suggestion
            "step": instance.data.get(
                "creator_attributes", {}
            ).get("step", 1.0), # missing
```

Do we still need to do something about this `missing`?

Reply:
```suggestion
            "step": instance.data.get(
                "creator_attributes", {}
            ).get("step", 1.0),
```